contracts/nns/contract.go

```diff
@@ -514,7 +514,12 @@ func SetRecord(name string, typ recordtype.Type, id byte, data string) {
 	ctx := storage.GetContext()
 	ns := getFragmentedNameState(ctx, tokenID, fragments)
 	ns.checkAdmin()
-	putRecord(ctx, tokenID, name, typ, id, data)
+	recordKey := getIdRecordKey(tokenID, name, typ, id)
```

Comment:
but why? both ways are ok to me but cannot agree with "it makes the contract code more complicated"
making a small description of a part of the code (making it a named func call) is not a complication to me

Reply:
These can add some confusion (similar names) and they're not reused.